app/controllers/admin/budget_investments_controller.rb

```diff
@@ -43,6 +52,41 @@ def toggle_selection
 
   private
 
+    def set_csv
+      attributes = [t("admin.budget_investments.index.table_id"),
+                    t("admin.budget_investments.index.table_title"),
+                    t("admin.budget_investments.index.table_supports"),
+                    t("admin.budget_investments.index.table_admin"),
+                    t("admin.budget_investments.index.table_valuator"),
+                    t("admin.budget_investments.index.table_geozone"),
+                    t("admin.budget_investments.index.table_feasibility"),
+                    t("admin.budget_investments.index.table_valuation_finished"),
+                    t("admin.budget_investments.index.table_selection")]
+      csv_string = CSV.generate(headers: true) do |csv|
+        csv << attributes
+        @investments.each do |investment|
+          id = investment.id.to_s
+          title = investment.title
+          total_votes = investment.total_votes.to_s
+          administrator = if investment.administrator.present?
+                            investment.administrator.name
+                          else
+                            t("admin.budget_investments.index.no_admin_assigned")
+                          end
+          valuators = if investment.valuators.empty?
+                        t("admin.budget_investments.index.no_valuators_assigned")
+                      else
+                        investment.valuators.collect(&:description_or_name).join(', ')
+                      end
+          heading_name = investment.heading.name
+          price = t("admin.budget_investments.index.feasibility.#{investment.feasibility}", price: investment.formatted_price)
```

Comment:
Could we do the interpolation outside of the `i18n.t` method call? So i18n-tasks will be able to check this string is in use?

Reply:
In admin.yml, the translation admin.budget_investments.index.feasibility includes at the end (%{price})".
Should I change the yml file?

Reply:
No, what I mean is that instead of:

Reply:
Oh, you're right. I'll change it